frontend post (#356)
ritterim/ritterim.github.io: _posts/2019-03-06-welcome-new-hire-getting-started-in-frontend-at-rimdev.md

---
layout: post
title: "Welcome New Hire: Getting started in Frontend at RIMdev"
date: 2019-03-06 11:09:18
tags:
- Team
- RIMdev
- Frontend
categories: Team
twitter_text: "Welcome New Hire: Getting started in #Frontend at RIMdev"
authors: Kevin Hougasian
image: https://images.unsplash.com/photo-1524530514346-eab43224489b?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2800&q=80
image_url: https://unsplash.com/photos/AqrD5QQ0ajM
image_credit: "@ilumire"
---

New to the frontend at RIMdev, or looking for more about our Frontend team? Here's a quick overview of our team, tools, and culture!

As a whole, RIMdev has always strived to use the technology that bets suites the task. So while this will give the current overview, software changes and so do we.

The majority of our frontend team are Mac users, so this post will seem mac-biased, but rest assured, that's where it stops. Use whatever platform that makes you happy.

> Some of this content may be mirrored in @buhakmeh's post [Welcome New Hire: Getting Started At RIMdev

Author: @mentions converts the twitter link automatically

- [Hugo](https://gohugo.io/getting-started/quick-start/): Go lang based static site generator.
- [VS Code](Visual Studio Code - Code Editing. Redefined
https://code.visualstudio.com/), [Atom](https://atom.io/), other?
- [iTerm2](https://www.iterm2.com/) or "[Oh My ZSH!](https://ohmyz.sh/)"
- [.NET Core SDK](https://dotnet.microsoft.com/download)

## Supporting cast

- [Lastpass](https://www.lastpass.com/)
- [Highfive](https://highfive.com): Video conferencing.
- [Spotify](https://spotify.com) Premium 😏
- [Headphones](https://www.cnet.com/topics/headphones/products/)

## That's really great, but what about professional development (me)?

### Conferences

Conferences are great opportunities to learn, be inspired, or inspire others by speaking! [VueConf](http://www.vueconf.us/) and [An Event Apart](https://aneventapart.com/) have become 2 of our top choices to date.

### Writing

As a developer or designer, you're faced with unique challenges every day -- so are your peers. [Write about them!](https://rimdev.io) Share your solution with the community. None of us work in a vacuum, and how you approach a solution through code or design can help everyone.

### Open Source

> We love Open Source, from [things we've made](https://rimdev.io/open-source/), to contributing to community projects and [Hacktoberfest](https://hacktoberfest.digitalocean.com/). Everyone on the team uses their personal GitHub account to build your community footprint. Some of our projects are private, although we keep many public where we can so your contributions are yours.

Reviewer: ```suggestion
We love Open Source, from [things we've made](https://rimdev.io/open-source/), to contributing to community projects and [Hacktoberfest](https://hacktoberfest.digitalocean.com/). Everyone on the team uses their personal GitHub account to build your community footprint. Some of our projects are private, although we keep much public in an effort to show your contributions.
```

### Ongoing

> Frameworks everywhere and all the things changing. Our landscape changes constantly, so we adapt and learn all the things. Instructor led training, subscriptions, whatever you need to succeed; [Pluralsight](https://www.pluralsight.com/), [egghead.io](https://egghead.io/), [Vue Mastery](https://www.vuemastery.com/), all there for the asking.

Reviewer: ```suggestion
Our professional landscape changes constantly, so we adapt and learn all the things. We all learn differently so we make many options available: Instructor-led training, book reimbursements, subscriptions, whatever you need to succeed. Some examples of learning tools include [Pluralsight](https://www.pluralsight.com/), [egghead.io](https://egghead.io/), [Vue Mastery](https://www.vuemastery.com/).
```

## Conclusion

> If being a part of a diverse, transparent, growing team while growing your career is what you're seeking, [Ritter Insurance Marketing's](https://ritterim.com) development team is a great environment. Team members to help you grow and face new challenges! If you're new the team, then Welcome! If you'd like to find out more, reach out on [twitter](https://twitter.com) 😃.

Reviewer: ```suggestion
If being a part of a diverse, transparent, and growing team while growing your own career is what you're seeking, [Ritter Insurance Marketing's](https://ritterim.com) development team is a great environment. Team members are here to help you grow and face new challenges! If you're new the team, then Welcome! If you'd like to find out more, reach out to any of us on [Twitter](https://twitter.com) 😃and if you're new, don't be afraid to start a conversation with your new team members.
```
